test/Github/Tests/HttpClient/PathPrependTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Github\Tests\HttpClient\Plugin;
+
+use Github\HttpClient\Plugin\PathPrepend;
+use GuzzleHttp\Psr7\Request;
+
+/**
+ * @author Nils Adermann <naderman@naderman.de>
+ */
+class PathPrependTest extends \PHPUnit_Framework_TestCase
```

Comment:
Travis complains. We should use PHPUnit 6

Reply:
Yes, just extend from `\PHPUnit\Framework\TestCase`. This works in both phpunit 5 and 6